### ISSUE TITLE
Distribution test build failures on Windows

### ISSUE THREAD

**Issue by bentoi** [zeroc-ice/freeze#3]
The C++ tests are still referencing 3.7.1 NuGet packages for Freeze:
```
<packages>
  <package id="zeroc.freeze.v120" version="3.7.1" targetFramework="native" />
  <package id="zeroc.freeze.v140" version="3.7.1" targetFramework="native" />
  <package id="zeroc.ice.v120" version="3.7.2" targetFramework="native" />
  <package id="zeroc.ice.v140" version="3.7.2" targetFramework="native" />
  <package id="zeroc.icebuilder.msbuild" version="5.0.4" targetFramework="native" />
</packages>
```

What's the plan for Freeze and 3.7.2? Are we releasing Freeze 3.7.2 or keeping 3.7.1 tested on top of Ice 3.7.2?  Right now the source tree builds 3.7.2 versioned binaries. AFAICT, the only source changes were to fix Windows build warnings.

**Comment by bernardnormier** [comment]
We'll release Freeze 3.7.2.

**Comment by pepone** [comment]
The failures was due to freeze NuGet package version not correctly updated, packages.config was referencing freeze 3.7.1 and project files were referencing 3.7.2

This cause restore to install 3.7.1 package and the build fail because the project expect 3.7.2

Fixed in 924f29ae29a653dec17a2a385448c763d452e7c1

I also fix dist-utils to correctly handle the version update in  this case
